charts/kyverno-policies/templates/cluster/cluster-job-ttl.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  annotations:
+    policies.kyverno.io/title: Set default TTL on Jobs that haven't specified it
+    policies.kyverno.io/category: Workarounds
+    policies.kyverno.io/description: >-
+      Ensure all completed and errored jobs do not remain in containerd with a
+      maximum age time to live.
+  name: cluster-job-ttl
+spec:
+  background: false
+  failurePolicy: Ignore
+  validationFailureAction: enforce
```

Comment:
@mitchty , Just wanted to check is it a good idea to have a separate policies for mutation and validation? Also any reason to make validationFailureAction is enforce instead of audit? Below we already have a mutation logic.

Reply:
When you have mutation logic below, we may not required validationFailureAction:enforce in the same policy file. We can refer upstream policy @ https://kyverno.io/policies/other/add-ttl-jobs/add-ttl-jobs/